archive_viewer/mixins/traces_table.py

```diff
@@ -89,6 +91,27 @@ def curve_delegates_init(self) -> None:
         delete_row_del = DeleteRowDelegate(self.ui.traces_tbl)
         self.ui.traces_tbl.setItemDelegateForColumn(delete_col, delete_row_del)
 
+    def dragEnterEvent(self, e):
```

Comment:
can you add a short docstring to this method?

Reply:
And include typhinting:
```suggestion
    def dragEnterEvent(self, e: QDragEnterEvent) -> None:
```

Reply:
I'll do this, but that would require importing QDragEnterEvent which isn't used anywhere other than to declare that the param will be of that type. yes?